scheduling/allocator.rb

```diff
@@ -262,6 +264,9 @@ def calculate_score
       # penalty for AX161, TODO: remove after migration to AX162
       score += 0.5 if @candidate_host[:total_cores] == 32
 
+      # penalty for hosts that do not accept slices if the request wants to use one
+      score += 1 if @request.use_slices && !@candidate_host[:accepts_slices]
```

Comment:
would that still result in a valid placement (putting a request that wants to use slice on a host that doesn't accept slices?